Tests/Tests/CDTReplicationTests.m

```diff
@@ -33,6 +64,44 @@ @interface CDTReplicationTests : CloudantSyncTests
 
 @implementation CDTReplicationTests
 
+- (void)setUp { [super setUp]; }
```

Comment:
if setup and teardown don't do anything additional then there's no point over-riding them (I guess this might have had debugging code or printfs in before)